tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot

```diff
@@ -12,9 +12,11 @@ Create container with port mappings
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start webserver
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  wget ${vch-ip}:10000 --retry-connrefused --timeout=10 --tries=10
+    ${rc}  ${output}=  Run And Return Rc And Output  wget ${vch-ip}:10000 --retry-connrefused --timeout=10
+    Log To Console  ${output}
```

Comment:
`log.html` truncates `${output}` in this case. I added `Log To Console` to display the complete output and see if nginx isn't up even after retrying several times - which will probably be the only reason this test fails. I can, however, take it out altogether - the logging statement is purely for diagnostic purposes.